app/src/main/java/agency/five/codebase/android/movieapp/MovieApp.kt

```diff
@@ -1,25 +1,34 @@
 package agency.five.codebase.android.movieapp
 
 import agency.five.codebase.android.movieapp.data.di.dataModule
+import agency.five.codebase.android.movieapp.data.di.databaseModule
+import agency.five.codebase.android.movieapp.data.di.networkModule
 import agency.five.codebase.android.movieapp.ui.favorites.di.favoritesModule
 import agency.five.codebase.android.movieapp.ui.home.di.homeModule
 import agency.five.codebase.android.movieapp.ui.moviedetails.di.movieDetailsModule
 import android.app.Application
 import android.util.Log
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidLogger
 import org.koin.core.context.startKoin
+import org.koin.core.logger.Level
 
 class MovieApp : Application() {
     override fun onCreate() {
         super.onCreate()
 
-        Log.d("MovieApp", "App started")
         startKoin {
+            androidLogger(Level.INFO)
+            androidContext(this@MovieApp)
             modules(
                 dataModule,
                 favoritesModule,
                 movieDetailsModule,
-                homeModule
+                homeModule,
+                networkModule,
+                databaseModule
             )
         }
+        Log.d("MovieApp", "App started")
```

Comment:
You should remove logs that you used for debugging before pushing/merging your work.